sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -388,6 +411,49 @@ make_buffer(
 Construct a SYCL buffer instance from a pointer to a Level Zero memory allocation. Please refer to <code>make_buffer</code>
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
+</tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          image<Dimensions, AllocatorT>> &,
+    const context &Context, event AvailableEvent = {})
+```
+</td>
+<td>This API is available starting with revision 4 of this specification.
+
+Construct a SYCL image instance from a ze_image_handle_t. 
+
+Because LevelZero has no way of getting image information from an image, it must be provided.  The  <code>backend_input_t</code> is a struct type like so:
+``` C++
+struct type {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    sycl::range<Dimensions> Range;
+    ext::oneapi::level_zero::ownership Ownership{
+        ext::oneapi::level_zero::ownership::transfer};
+  };
+```
+where the Range should be ordered (width), (width, height), or (width, height, depth) for 1D, 2D and 3D images respectively, 
+with those values matching the dimensions used in the `ze_image_desc` that was used to create the `ze_image_handle_t` initially. 
+
+Example Usage
+``` C++
+sycl::backend_input_t<BE, sycl::image<2>> ImageInteropInput{ ZeHImage, ChanOrder, ChanType, ImgRange_2D, sycl::ext::oneapi::level_zero::ownership::transfer };      
+    
+auto Image_2D  = sycl::make_image<BE, 2>(ImageInteropInput, Context);

```

Comment:
```suggestion
auto Image_2D  = sycl::make_image<BE, sycl::image<2>>(ImageInteropInput, Context);
```

I think, right?

Reply:
No.  It is correct as written.  I could lose the `auto` though.  That might make it clearer.
`sycl::image<2> Image_2D = sycl::make_image<BE, 2>(ImageInteropInput, Context);`

Reply:
I see.  I think it would help if you changed the prototype of `make_image` in the left column of this table to show the template parameters.  For example:

```
template<backend Backend, int Dimensions = 1, typename AllocatorT = sycl::image_allocator>
image<Dimensions, AllocatorT> make_image(
    const backend_input_t<backend::ext_oneapi_level_zero,
                          image<Dimensions, AllocatorT>> &,
    const context &Context, event AvailableEvent)
```

I'm assuming you have a template parameter `AllocatorT` like this?  I also assume the `Dimensions` template parameter defaults to `1` to match the SYCL 1.2.1 spec, where the `image` object also defaults to 1 dimension?